### PR TITLE
changes to composer.json for change in the namespace Rhumsaa

### PR DESCRIPTION
**Pull request by ShekharReddy4**

### CODE REVIEW

**Comment by psbrandt** [comment]
Did this end up being the only change you need to make to get the server up and running?

File: composer.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
 	"license": "MIT",
 	"require": {
 		"laravel/framework": "4.1.*",
-		"rhumsaa/uuid": "~2.7",
 		"doctrine/dbal": "2.4.*",
-		"phpunit/phpunit": "4.0.*"
```
Comment (psbrandt) [inline composer.json]: Was `phpunit` causing issues?

Comment (ShekharReddy4) [inline composer.json]: No, This is a formatting issue. Do I need to commit again? :)

**Comment by ShekharReddy4** [comment]
@psbrandt 
No, this doesn't solve the issue I stuck with, but this resolves the error when we try to install the dependencies. the error might look something like this "Package rhumsaa/uuid is abandoned, you should avoid using it. Use ramsey/uuid instead."

**Pull request merged**